pebbles/drivers/provisioning/docker_driver.py

```diff
@@ -251,6 +251,7 @@ def log(self, param, info=False):
             options=options,
             passwords=None
         )
+        logger.debug('ResultCallback executed')
```

Comment:
Isn't that a bit premature as the callback is executed on the next line? More "Executing ResultCallback"?